sklearn/decomposition/fastica_.py

```diff
@@ -563,8 +565,7 @@ def inverse_transform(self, X, copy=True):
         """
         check_is_fitted(self, 'mixing_')
 
-        if copy:
```

Comment:
so even if ``self.mixing_`` is float, ``fast_dot(X, self.mixing_.T)`` is not? That is somewhat surprising to me.